ios-sdk/TangramVC.swift

```diff
@@ -12,6 +12,7 @@ class TangramVC:  TGMapViewController{
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.queueSceneUpdate("sources.mapzen.url_params.api_key", withValue: "vector-tiles-7bfyfz8")
```

Comment:
Looking at https://github.com/mapzen/ios/blob/master/tangram/scenes/scene.yaml#L384-L393 I think the issue is that we need to update to the latest version of bubble-wrap, which has sources that look like this https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L537-L548 - after that this line should work perfectly!